docs/Plugin Architecture/Plugin Management.md

```diff
@@ -93,14 +93,14 @@ To prevent duplication, this definition has been moved [here][plugin.json format
 
 ##### Command Usage
 ```commandline
-cli plugin install [plugin...] [--registry <registry>] [--isLocalURI] [--isRemoteURI] [--use <config>]
+zowe plugins install [plugin...] [--registry <registry>] [--isLocalURI] [--isRemoteURI] [--use <config>]
 ```
 - `[plugin...]` is an npm package or a pointer to a URI (either local or remote). It is passed the same way it would be to npm install. 
-    - Semantic versioning should also be handled for each plugin much like how it is done in `npm install`. So versions for plugins could be specified like `cli plugin install plugin@^1.0.0` and this would be saved in the plugin.json file so that the update command can respect this.
+    - Semantic versioning should also be handled for each plugin much like how it is done in `npm install`. So versions for plugins could be specified like `zowe plugins install plugin@^1.0.0` and this would be saved in the plugin.json file so that the update command can respect this.
```

Comment:
- plug-in, plug-ins
- please use backticks for the json file

Reply:
In this case, we are giving an example of the command to run in the command line. For the CLI, the plugins command group is not hyphenated. Will add the backticks.